addon/authenticators/osf-token.js

```diff
@@ -40,6 +40,11 @@ export default BaseAuthenticator.extend({
     },
     authenticate(accessToken) {
         // Adds the entire API user endpoint record to the session store as given
-        return this._test(accessToken);
+        let jqDeferred = this._test(accessToken);
```

Comment:
Yes... but jquery promise semantics are slightly different from the ones ember is built on. This causes trouble when ember internals (like the beforeModel hook) try to deal with, eg, failure states.

(eg jquery promises lack a `.catch`)

It's quite ridiculous, actually. See, eg https://stackoverflow.com/questions/18808079/can-jquery-promises-be-chained-with-ember-rsvp-promises